WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -144,9 +144,9 @@ def alert(type, content = nil, note: false, &block)
     content_tag :div, content, class: "alert alert-#{type}"
   end
 
-  def users_to_sentence(users, include_email: false)
+  def users_to_sentence(users, include_profile: false)
     "".html_safe + users.sort_by(&:name).map do |user|
-      include_email ? mail_to(user.email, user.name) : user.name
+      include_profile && user.wca_id ? link_to(user.name, person_path(user.wca_id)) : user.name
```

Comment:
The tests are failing because `user.name` is a plain string and this causes trouble when concatenating with htmlsafe strings. This must've been missed because before we either had a list of links or a list of plain strings ^^ Using `html_escape(user.name)` should fix it!

Reply:
Ok thanks, I wasn't sure why it was getting HTML encoded!

Reply:
Apparently the mailer templates don't seem to have `html_escape` imported, I'm not really sure why, but using the fully qualified name does help: `ERB::Util.html_escape(user.name)`.